docs/content/v2024.2/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -143,6 +143,20 @@ Optionally, use the **Helm Overrides** section, as follows:
 
     To enable the GKE service account service at the provider level, refer to [Overrides](../../configure-yugabyte-platform/kubernetes/#overrides).
 
+- If you want to enable readiness probes, add the following overrides:
```

Comment:
You can just link to the OSS "readiness probes" here and remove the statement -For more information, refer to [Readiness probes]..